src/__test__/components/Collapsible/Collapsible.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import Collapsible from "@components/Collapsible";
+import { render, screen } from "@testing-library/react";
+import '@testing-library/jest-dom'
+
+describe('Collapsible', () => {
+    test('Checking collapsible title', () => {
+        render(<Collapsible title={'Teste'} />);
+        const button = screen.getByRole('button', { name: 'Teste' });
+
+        expect(button).not.toBeDisabled();
+        expect(button).toHaveTextContent('Teste');
+    });
+
+    test('Checking if collapsible is opened', () => {
+        const collapsible = render(<Collapsible title={'Teste'} />);
+
+        const button = collapsible.getByRole('button');
+        expect(button.className).toEqual('Collapsible__trigger is-open');
```

Comment:
**suggestion (blocker):** please add a content inside the component and verify if this content exist instead of verify the html props behavior